### ISSUE TITLE
sql: better document the session variable for transaction priority 

### ISSUE THREAD

**Issue by awoods187** [cockroachdb/docs#8462]
We support session variables for transaction priorities:
```SQL
root@127.0.0.1:57215/movr> SET SESSION CHARACTERISTICS AS TRANSACTION PRIORITY high;
SET
```
But you can't easily find this information from a search of the docs:
![image](https://user-images.githubusercontent.com/22278911/94089384-0a453c80-fde1-11ea-893b-6edc33ed62f0.png)

Instead, it directs you to the transaction priority [docs that cover per transaction setting](https://www.cockroachlabs.com/docs/v20.1/set-transaction.html). In fact, those docs mention the session variable `transaction_priority`. This is confusing because that variable is for _showing_ the sessions transaction priority not for the changing it:
```SQL
set transaction_priority = 'high';
ERROR:  unrecognized configuration parameter "transaction_priority"
```
```SQL
show transaction_priority;
  transaction_priority
------------------------
  high
(1 row)
```

We should make it easier for developers to find and use this information.

**Comment by jseldess** [comment]
@ericharmeling, @vy-ton, to me, this supports the notion that we should have a "Working with Transactions" section in our developer guide. I think we may be able to break the topic into more task-oriented topics.

**Comment by awoods187** [comment]
So I may have jumped the gun. The [PR](https://github.com/cockroachdb/cockroach/pull/47641) that added this feature merged in April so I assumed it was in 20.1 but it was delayed to 20.2 due to the stability period. That being said, I like the idea of a "working with transactions" section in the developer guide and we should clarify the show guidance.
